source/content/guides/integrated-composer/03-private-repo-package.md

```diff
@@ -33,6 +37,16 @@ For this procedure, a GitHub token will be added to your code repository. It all
         }
     ],
     ```
+    
+    GitLab syntax: 
+    ```json:title=composer.json
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://oauth2:<token>@gitlab.com/mycompany/my-private-repo.git"
+        }
+    ],
+    ```
 
 1. Run the command below to require the package and specify the branch, prefixed with `dev-`
```

Comment:
"Run the command below" but there's no command here. There's sample lines of what should be modified in the "composer.json" file, but that's not a command.

It looks like the command should be this:

composer require mycompany/my-private-repo:dev-branch-name